index.njk

```diff
@@ -1,8 +1,5 @@
 ---
 layout: layouts/home.njk
-eleventyNavigation:
```

Comment:
Having "Home" in the navigation was duplicative of the link in the "Free Free Api Education" link. The Home link was also causing the navigation items to wrap to a second line at the narrower width. I removed Home from the navigation to address these 2 issues.